setup.py

```diff
@@ -0,0 +1,73 @@
+from importlib.machinery import SourceFileLoader
+import io
+import os.path
+
+from setuptools import find_packages, setup
+
+sourcedml = SourceFileLoader("sourced-ml-core", "./sourced/ml/core/__init__.py").load_module()
+
+with io.open(os.path.join(os.path.dirname(__file__), "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
+tf_requires = ["tensorflow>=1.0,<2.0"]
+tf_gpu_requires = ["tensorflow-gpu>=1.0,<2.0"]
+exclude_packages = (
+    ("sourced.ml.core.tests", "sourced.ml.core.tests.source")
+    if not os.getenv("ML_CORE_SETUP_INCLUDE_TESTS", False)
+    else ()
+)
+
+
+setup(
+    name="sourced-ml-core",
+    description="Framework for machine learning on source code. "
```

Comment:
We should change this: this is not a framework anymore

Reply:
I changed it to
>Library containing the core algorithms for machine learning on source code.